fun/draw.cpp

```diff
@@ -36,10 +32,16 @@ int setupTerm() {
   if (tcsetattr(0, TCSAFLUSH, &newtermios) < 0) {
     return -1;
   }
+
+  // Switch to alternate screen mode.
+  printf("\033[?1049h");
```

Comment:
Neat :)  I kind of like the residue being left, but interesting to take this in more of a first-class app ;)